esp_encrypted_img/idf_component.yml

```
@@ -0,0 +1,6 @@
+dependencies:
+  idf:
+    version: '>=5.0'
```

```suggestion
    version: '>=4.3'
```

This is standalone component, should work with previous IDF releases as well?